docs/advanced/runtime/docs/ascii-strings.md

```diff
@@ -0,0 +1,127 @@
+  - /Mono:Runtime:Documentation:AsciiStrings/
+---
```

Comment:
for context: Jekyll (the static site generator that processes the website) requires a so called [front matter](https://jekyllrb.com/docs/frontmatter/) block.